pkg/crypto/tinkcrypto/primitive/jwe/ecdhes/ecdhes.go

```diff
@@ -0,0 +1,74 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+// Package ecdhes provides implementations of JWE encryption using ECDH-ES KW primitives.
+//
+// The functionality of ecdhes Encryption is represented as a pair of
+// primitives (interfaces):
+//
+//  * ECDHESEncrypt for encryption of data as JWE
+//
+//  * ECDHESDecrypt for decryption of JWE data and returning decrypted plaintext
```

Comment:
There should not be explicit JWE data. JWE is a higher layer.

Reply:
JWE reference is now removed from this change. It will be introduced in a another PR specifically for JWE outside of crypt package.